src/Catty/DataModel/Scripts/WhenBackgroundChangesScript.m

```diff
@@ -0,0 +1,88 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+#import "WhenBackgroundChangesScript.h"
+#import "CBMutableCopyContext.h"
+
+@implementation WhenBackgroundChangesScript
+
+- (kBrickCategoryType)category
+{
+    return kControlBrick;
+}
+
+- (NSString*)pathForLook
+{
+    return [NSString stringWithFormat:@"%@%@/%@", [self.object projectPath], kProjectImagesDirName, self.look.fileName];
```

Comment:
The path to the looks has changed. Please use `[look pathForScene]` instead.

Reply:
Thank you for the hint! I changed it.